certified-connectors/Docurain/apiDefinition.swagger.json

```diff
@@ -0,0 +1,248 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Docurain",
+    "description": "DocurainAPI Communication Connector. Docurain API allows you to create your own documents using template files registered in Docurain and data from PowerAppsn.",
+    "contact": {
+      "name": "Docurain support",
+      "url": "https://api.docurain.jp/console/inquiry",
+      "email": "cs@docurain.jp"
+    }
+  },
+  "host": "powerapps.docurain.jp",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/apiw/{outType}/{templateName}": {
+      "post": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "title": "body",
+              "type": "string",
+              "format": "binary",
+              "description": "If the process is successful, the binary data of the generated form is returned. If the process fails, JSON with the details of the error is returned."
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "Render",
+        "operationId": "Render",
+        "description": "This API renders a form by specifying the name of a template saved in Docurain beforehand.",
+        "parameters": [
+          {
+            "name": "outType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Set the output format: pdf, xls(xlsx), png, jpg, jpg, gif, svg",
+            "x-ms-summary": "output format"
+          },
+          {
+            "name": "templateName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify the name of the template you have already saved in Docurain.",
+            "x-ms-summary": "pre-saved template name"
+          },
+          {
+            "name": "X-Docurain-Data-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Set the data format : json, csv, tsv",
+            "x-ms-summary": "data format"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "description": "Specify the string data in the format (json/csv/tsv) that you specify for the data format.",
+            "schema": {
+              "type": "string",
+              "description": "Specify the format of the data in the data format.",
+              "title": "data",
+              "format": ""
+            }
+          }
+        ]
+      }
+    },
+    "/apiw/async/{outType}/{templateName}": {
+      "post": {
+        "responses": {
+          "default": {
```

Comment:
@geckoR42 Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
@sriyen-msft Removed "default". Fixed.